Microsoft.Toolkit.Uwp.UI/Helpers/Emoji/EmojiManager.cs

```diff
@@ -0,0 +1,391 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System.Globalization;
+using System.Linq;
+
+namespace Microsoft.Toolkit.Uwp.UI.Helpers.Emoji
+{
+    /// <summary>
+    ///     Class to help manage use of Emojis
+    /// </summary>
+    public static class EmojiManager
+    {
+        private static readonly EmojiDetails[] EmojiArray =
+        {
+            new EmojiDetails(Emoji.HelmetWithWhiteCross, new[] {0x26D1, 0xFE0F}, true), new EmojiDetails(Emoji.Kiss, new[] {0x1F48F}, true), new EmojiDetails(Emoji.CoupleWithHeart, new[] {0x1F491}, true),
```

Comment:
Well, we do it like we always did: someone opens an issue complaining about the missing emojis, and then someone provides a PR with the new ones!

Might be useful to provide a manual process to set an emoji, when not included in this list though! Alternatively, make this a `List<EmojiDetails>` so anyone can manually add new emojis in their projects!